src/main/java/nl/activakingdoms/wars/Handler.java

```diff
@@ -27,5 +32,25 @@ public void onAttack(EntityDamageByEntityEvent e) {
         }
     }
 
+    @EventHandler
+    public void onBlockBreak(BlockBreakEvent e) {
+
+        War war = GeneralMethods.getWar();
+
+        Player player = e.getPlayer();
+        if (war.containsPlayer(player)) {
+            ArrayList<Player> notificationList = new ArrayList<>();
```

Comment:
Dus voor elk nieuw blok dat weggehaald wordt maken, rekenen we alles opnieuw uit. Is het niet efficiënter om notificationList bij aanmaken van war aan te maken en pas bij nieuwe removal te updaten?

Reply:
maar in beide versies zal je hem updaten bij een blockbreak
en ik weet niet hoe ik een andere manier van updaten zou kunnen doen, maar als jij die hebt is dat idd wel sneller
maar als er geen andere manier van updaten komt dan is dit even snel

Reply:
notificationList verandert toch alleen wanneer iemand dit zelf toggled? Dus het lijkt mij dat je m dan daar kan updaten

Reply:
Nee want als iemand offline was tijdens "/war setup" dan wordt de list aangemaakt zonder die speler (ik krijg ze via Bukkit.getOnlinePlayers() anders zouden we door alle OfflinePlayer's moeten loopen)

Of we geven de helper rol even die notification permission nadat de war is opgezet.
Dus dan zou je eens in de zoveel tijd moeten checken wie er allemaal die permission heeft.

Al zouden we in de honderden spelers hebben zou t miss op een andere manier doen, misschien dit in een losse Thread doen, ik hoop dat jij daar dan goed in bent want ik kan t niet.

Reply:
We kunnen misschien ergens een staff playerlist bijhouden, dat we daar doorheen loopen, dat scheelt al